From: Colin Ian King <colin.king@canonical.com>
Date: Thu, 24 Aug 2017 09:25:53 +0100
Subject: RDMA/qedr: fix spelling mistake: "invlaid" -> "invalid"
Patch-mainline: v4.14-rc1
Git-commit: 847cb1a3566ca5b47b94a8e90f881b533f35cf07
References: bsc#1050545 FATE#322893

Trivial fix to spelling mistake in DP_ERR error message

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Reviewed-by: Leon Romanovsky <leonro@mellanox.com>
Reviewed-by: Ram Amrani <Ram.Amrani@cavium.com>
Signed-off-by: Doug Ledford <dledford@redhat.com>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/infiniband/hw/qedr/verbs.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/infiniband/hw/qedr/verbs.c
+++ b/drivers/infiniband/hw/qedr/verbs.c
@@ -491,7 +491,7 @@ struct ib_pd *qedr_alloc_pd(struct ib_de
 		 (udata && context) ? "User Lib" : "Kernel");
 
 	if (!dev->rdma_ctx) {
-		DP_ERR(dev, "invlaid RDMA context\n");
+		DP_ERR(dev, "invalid RDMA context\n");
 		return ERR_PTR(-EINVAL);
 	}
 
